src/core/i18n/sv/translation.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
Everything in this file is still 'en' rather than 'sv'.

Reply:
To be replaced with actual translations